### PR TITLE
0.1.0 Assimilation to Rally

### PR DESCRIPTION
**Pull request by jeffmay**
@patrick-bohan @Ackertron @usufruct99

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+language: scala
```
Comment (usufruct99) [inline .travis.yml]: (wat)

**Comment by usufruct99** [comment]
lgtm

**Comment by patrick-bohan** [comment]
How does this relate to https://github.com/AudaxHealthInc/lib-argosy-sigaldry ?

**Comment by patrick-bohan** [comment]
@slustbader @petercoons @ripplekhera

File: .travis.yml (hunk 1 above)
Comment (petercoons) [inline .travis.yml]: just making sure :trollface:

**Comment by paulpdaniels** [comment]
Will this work supersede or replace some of the work that @scottrice10 is doing on AudaxHealthInc/lib-wellness-ws-util#26 and AudaxHealthInc/lib-zenplay-commons#48 to pass through locale information for i18n?

File: build.sbt [hunk 2]
```diff
@@ -1,10 +1,17 @@
...
-  scalaVersion := "2.11.7"
+  organization := "com.rallyhealth",
+  organizationName := "Rally Health",
+  scalaVersion := "2.11.7",
+
+  // disable compilation of ScalaDocs, since this always breaks on links and isn't as helpful as source
+  sources in(Compile, doc) := Seq.empty,
+  // disable publishing empty ScalaDocs
+  publishArtifact in (Compile, packageDoc) := false,
+  licenses += ("Apache-2.0", url("http://opensource.org/licenses/apache-2.0"))
```
Comment (patrick-bohan) [inline build.sbt]: This seems like something that should be in upstream.

File: project/build.properties [hunk 3]
```diff
@@ -1 +1 @@
-sbtVersion=0.13.9
+sbt.version=0.13.9
```
Comment (patrick-bohan) [inline project/build.properties]: Bug upstream?

**Comment by patrick-bohan** [comment]
LGTM

**Comment by jeffmay** [comment]
@paulpdaniels I think this will fit very well with that work. This will allow you to create a WSClient that requires an implicit locale be in scope. You can probably move this over to using this library with little friction.

**Comment by jeffmay** [comment]
retest this please

**Comment by jeffmay** [comment]
retest this please

**Comment by jeffmay** [comment]
gonna try reopenning this PR to trigger the Jenkins build.

**Comment by jeffmay** [comment]
retest this please

**Comment by Ackertron** [comment]
retest this please

**Comment by Ackertron** [comment]
retest this please

**Comment by Ackertron** [comment]
retest this please

**Comment by jeffmay** [comment]
retest this please

**Comment by jeffmay** [comment]
Released as v0.1.0: http://jenkins.infra.careverge.com:8080/view/By%20Category/view/Release/job/Release_scala-context-passing/1/

**Pull request merged**